bin/lisky

```diff
@@ -16,6 +16,9 @@
  */
 var os = require('os');
 var lockfile = require('lockfile');
+var semver = require('semver');
+
+var packagejson = require('../package.json');
```

Comment:
Prefer capitals for acronyms. See https://github.com/LiskHQ/lisky/pull/401/files